src/test/java/com/wikia/webdriver/testcases/visualeditor/VEImageTests.java

```diff
@@ -0,0 +1,109 @@
+package com.wikia.webdriver.testcases.visualeditor;
+
+import com.wikia.webdriver.common.contentpatterns.PageContent;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.dataprovider.VisualEditorDataProvider.Alignment;
+import com.wikia.webdriver.common.dataprovider.VisualEditorDataProvider.Setting;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.visualeditordialogs.VisualEditorAddMediaDialog;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.visualeditordialogs.VisualEditorMediaSettingsDialog;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.visualeditordialogs.VisualEditorReviewChangesDialog;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.visualeditordialogs.VisualEditorSaveChangesDialog;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.article.ArticlePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.visualeditor.VisualEditorPageObject;
+
+import org.joda.time.DateTime;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class VEImageTests extends NewTestTemplate {
+  // AM05
+  @Test(groups = {"VEImageTests", "VEImageTests_001", "VEMediaPreview"})
+  @Execute(asUser = User.USER_9)
+  public void VEImageTests_001_previewImage() {
+    String mediaTitle = "Thomas Wright 1792 - 1849";
+
+    VisualEditorPageObject ve =
+        new VisualEditorPageObject(driver).openVEOnArticle(wikiURL, PageContent.ARTICLE_NAME_PREFIX
+                                                                    + DateTime.now().getMillis());
+    ve.verifyVEToolBarPresent();
+    ve.verifyEditorSurfacePresent();
+    VisualEditorAddMediaDialog mediaDialog = ve.clickImageButton();
+    mediaDialog = mediaDialog.searchMedia(mediaTitle);
+    ve = mediaDialog.previewExistingMediaByTitle(mediaTitle);
+    ve.verifyPreviewImage();
+  }
+
+  // MS01
+  @Test(groups = {"VEImageTests", "VEImageTests_002", "VEMediaSetting"})
+  @Execute(asUser = User.USER_9)
+  public void VEImageTests_002_editCaption() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)